### PR TITLE
Advanced HTML and CSS Course: Adds A11y Project as Additional Resource

### LINKED ISSUE #27971: Advanced HTML and CSS Course: Add an additional resource to The Web Content Accessibility Guidelines (WCAG)
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

The additional resources page mentions it is seeking additional resources. 

> This section contains helpful links to related content. It isn’t required, so consider it supplemental.
> 
> -   It looks like this lesson doesn’t have any additional resources yet. Help us expand this section by contributing to our curriculum.

Suggested additional resources: 

> This section contains helpful links to related content. It isn’t required, so consider it supplemental.
> 
>    - [The A11Y Project:](https://www.a11yproject.com/) An open-source resource that provides information on how to make your web content more accessible. The A11Y Project includes checklists, guidelines, and tools that can help you implement WCAG standards effectively.
> - [Introduction to Web Accessibility by W3Cx (edX](https://www.edx.org/learn/web-accessibility/the-world-wide-web-consortium-w3c-introduction-to-web-accessibility)): This free course, created by the World Wide Web Consortium (W3C) and offered on edX, provides an introduction to web accessibility. It covers the principles of WCAG and practical steps for creating accessible web content.

### Path

Ruby / Rails, Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/advanced-html-and-css-the-web-content-accessibility-guidelines-wcag

### (Optional) Discord Name

_No response_

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by kellyky**
## Because

Resolve issue - to add an additional resource to this lesson. 

## This PR

Adds to Additional Resources:

[The A11Y Project:](https://www.a11yproject.com/) An open-source resource that provides information on how to make your web content more accessible. The A11Y Project includes checklists, guidelines, and tools that can help you implement WCAG standards effectively.

## Issue

Closes #27971 

## Additional Information

n/a

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [ ] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Thanks @kellyky 
On top of the nit below, there are also a few simple lint errors - would you be happy to fix these as well? They should fixable with a single pass of our `fix:lesson` script as per our [local linting and fixing instructions](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).

File: advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md
Comment (kellyky) [reply advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md:66]: Good catch, thanks!

**Review by MaoShizhong — APPROVED** [review]
Are you happy to fix the few remaining lint errors? The error messages should be self explanatory, but we also have lint and fix instructions that I've linked in the message above.

If so, I'll wait for the lint errors to be fixed before merging.

**Comment by kellyky** [comment]
> Are you happy to fix the few remaining lint errors? The error messages should be self explanatory, but we also have lint and fix instructions that I've linked in the message above.
> 
> If so, I'll wait for the lint errors to be fixed before merging.

Sure - I should be able to later today or tomorrow morning!

**Comment by kellyky** [comment]
All set! (And all in all a good reminder for me to mind the linting check).

**Review by MaoShizhong — APPROVED** [review]
:rocket:

For future PRs, you if you're not sure then you don't need to handle lint errors if you haven't been asked to. You can always comment asking if a maintainer would like you to fix them, especially since some lint errors can only be fixed manually and are based on context.

**Pull request merged**
